src/mlpack/methods/mean_shift/mean_shift_impl.hpp

```diff
@@ -265,8 +265,9 @@ inline void MeanShift<UseKernel, KernelType, MatType>::Cluster(
   // forcing convergence, take 1 random centroid calculated.
   if (centroids.empty())
   {
-    Log::Warn << "No clusters converge, setting 1 random centroid calculated. "
-    "Try a larger max_iterations or pass force_convergence flag." << std::endl;
+    Log::Warn << "No clusters converged; setting 1 random centroid calculated. "
```

Comment:
The error message is really command line specific, perhaps a more general approach like ''increase the number of iterations or set force converge`` does work as well?

Reply:
Sure, that can work too.  I guess the parameter may not always be called `max_iterations` or `force_convergence`, depending on the language being used.  And a simple approach like what you suggested should work fine.  Fixed in bf2e7ba73.